batch/batch/cloud/gcp/resource_utils.py

```diff
@@ -8,6 +8,7 @@
 MACHINE_TYPE_REGEX = re.compile('(?P<machine_family>[^-]+)-(?P<machine_type>[^-]+)-(?P<cores>\\d+)')
 GCP_MACHINE_FAMILY = 'n1'
 
+GPU_VERSIONS = {"g2": "l4"}
```

Comment:
Is this `MACHINE_TYPE_TO_ACCELERATOR_VERSIONS`? What is the difference between a gpu and an accelerator? If they're the same, can we choose one or the other?

Reply:
They are the same I believe, I will update to use accelerator when referencing the GPU itself